dev/merge_arrow_pr.py

```diff
@@ -287,7 +311,7 @@ def version_tuple(x):
     mainline_versions = [v for v in mainline_versions
                          if f"maint-{v}" not in maintenance_branches]
     default_fix_versions = [
-        fix_version_from_branch(x, mainline_versions)
+        fix_version_from_branch(mainline_versions)
         for x in merge_branches]
 
     return default_fix_versions[0]
```

Comment:
As we are not using `x` the list comprehension is not needed anymore. We can update this to be:
```
return fix_version_from_branch(mainline_versions)
```
We can also remove the `merge_branches` variable definition as is not used anymore and remove it from the signature of the function.